tap_sftp/sync.py

```diff
@@ -64,7 +64,8 @@ def sync_file(sftp_file_spec, stream, table_spec, config, sftp_client):
             'delimiter': table_spec.get('delimiter', ','),
             'file_name': sftp_file_spec['filepath'],
             'encoding': table_spec.get('encoding', 'utf-8'),
-            'sanitize_header': table_spec.get('sanitize_header', False)}
+            'sanitize_header': table_spec.get('sanitize_header', False),
+            'clean_colnames': table_spec.get('clean_colnames', False)}
```

Comment:
```suggestion
            'skip_rows': table_spec.get('skip_rows', False)}
```
@radbrt should this be skip_rows vs clean_colnames?